.gitignore

```diff
@@ -12,6 +12,8 @@ web/.DS_Store
 web/public/.DS_Store
 web/src/.DS_Store
 
+database.cpython-39.pyc
+webscraper.cpython-39.pyc
 .db
 table.db
-__pycache__
\ No newline at end of file
+__pycache__
```

Comment:
This should be service/table.db and service/__pycache__ to stop those files from being entered in your commit in the first place.